contracts/interfaces/ILensHub.sol

```diff
@@ -90,14 +90,16 @@ interface ILensHub {
      * @notice Creates a profile with the specified parameters, minting a profile NFT to the given recipient. This
      * function must be called by a whitelisted profile creator.
      *
+     * @return The token ID of the profile which was created.
+     *
      * @param vars A CreateProfileData struct containing the following params:
      *      to: The address receiving the profile.
      *      handle: The handle to set for the profile, must be unique and non-empty.
      *      imageURI: The URI to set for the profile image.
      *      followModule: The follow module to use, can be the zero address.
      *      followModuleData: The follow module initialization data, if any
      */
-    function createProfile(DataTypes.CreateProfileData calldata vars) external;
+    function createProfile(DataTypes.CreateProfileData calldata vars) external returns (uint256 profileId);
```

Comment:
Hey @liamzebedee ! The CI check is failing because you are returning a named param `profileId` and it's not in the natspec.

So you can add it to the natspec, or just remove the name which also follows the convention of the rest of the code:
```suggestion
    function createProfile(DataTypes.CreateProfileData calldata vars) external returns (uint256);
```